apps/src/sites/studio/pages/congrats/index.js

```diff
@@ -43,7 +43,11 @@ $(document).ready(function () {
   const mcShareLink = tryGetLocalStorage('craftHeroShareLink', '');
 
   userType === 'teacher' &&
-    analyticsReporter.sendEvent(EVENTS.TEACHER_HOC_CONGRATS_PAGE_VISITED);
+    analyticsReporter.sendEvent(EVENTS.TEACHER_HOC_CONGRATS_PAGE_VISITED, {
```

Comment:
Should this be `TEACHER_VISITED_CONGRATS_PAGE` instead of `TEACHER_HOC_CONGRATS_PAGE_VISITED`?

Reply:
Example event:
```
[AMPLITUDE ANALYTICS EVENT]: Teacher Visited Congrats Page. Payload: {"payload":{"isHocTutorial":false,"isPlCourse":true,"isK5PlCourse":false}}
```